fourmolu.yaml

```diff
@@ -0,0 +1 @@
+indentation: 2
```

Comment:
I think we’d also prefer to keep beautifiers out of the tree, if only so as to avoid another years-long (and if I exaggerate, it is only slightly) argument between me and @robrix about the virtue (_or lack thereof_) of horizontal alignment. 😄